config/default/core.extension.yml

```diff
@@ -15,6 +15,7 @@ module:
   block_content_permissions: 0
   block_content_template: 0
   breakpoint: 0
+  ccom_core: 0
```

Comment:
This will turn it on for all sites. I'm guessing this was an accidental export when doing the name change stuff.

Reply:
Yeah, this should be removed and instead an entry to [config_ignore](config/default/config_ignore.settings.yml) should be added to ignore the feature's status.